components/schema-migrator/migrations/director/20210907181222_ord_extensive_querying.up.sql

```diff
@@ -0,0 +1,88 @@
+BEGIN;
```

Comment:
As the migrations are becoming more and more complex and long to check. Can you add as a comment above each view what is the change since the previous db schema

Reply:
I described it in the PR description, but probably for the long run, it could be useful to include the diff in the migration as well. 

We can sync with the team and probably introduce it as a practice for the bigger migrations from now on.